ui/views.py

```diff
@@ -117,42 +116,28 @@ class VideoDetail(TemplateView):
 
     def get_context_data(self, video_key, **kwargs):  # pylint: disable=arguments-differ
         context = super().get_context_data(**kwargs)
-        default_settings = json.loads(default_js_settings(self.request))
         video = get_object_or_404(Video, key=video_key)
-        context['video'] = video
-        context['form'] = forms.VideoForm(instance=video)
-        try:
-            videofile = video.videofile_set.get(encoding=EncodingNames.HLS)
-            default_settings['videofile'] = videofile.cloudfront_url
-        except VideoFile.DoesNotExist:
-            videofile = None
-            default_settings['videofile'] = None
-
-        context['videofile'] = videofile
-        context["js_settings_json"] = json.dumps(default_settings)
+        context["js_settings_json"] = json.dumps({
+            **default_js_settings(self.request),
+            'videoKey': video.key.hex,
+        })
```

Comment:
If the video is not done uploading or transcoding, the video player won't work (but will display a large black box) with javascript errors in the console.  The try..except block formerly here was used to decide whether or not to display 'Your video is still processing' or something like that instead of the video player.